b2sdk/sync/policy.py

```diff
@@ -305,6 +305,60 @@ class DownAndKeepDaysPolicy(DownPolicy):
     pass
 
 
+class CopyPolicy(AbstractFileSyncPolicy):
+    """
+    File is copied (server-side).
+    """
+    DESTINATION_PREFIX = 'b2://'
+    SOURCE_PREFIX = 'b2://'
+
+    def _make_transfer_action(self):
+        return B2CopyAction(
+            self._source_file.name,
+            self._source_folder.make_full_path(self._source_file.name),
+            self._source_file.latest_version().id_,
+            self._dest_folder.make_full_path(self._source_file.name),
+            self._get_source_mod_time(),
+            self._source_file.latest_version().size,
+        )
+
+
+class CopyAndDeletePolicy(CopyPolicy):
+    """
+    File is copied (server-side) and the delete flag is SET.
+    """
+
+    def _get_hide_delete_actions(self):
+        for action in super()._get_hide_delete_actions():
```

Comment:
What actions might the parent class have that we want to use here?  (It looks like the actual base class just returns the empty list.)

Reply:
If the parent class will start returning something, we'd like this class (and all descendants) to honor that without modification of the inheriting classes